Cargo.toml

```diff
@@ -29,7 +27,7 @@ moxin-protocol = { path = "moxin-protocol" }
 moxin-backend = { path = "moxin-backend" }
 moxin-fake-backend = { path = "moxin-fake-backend" }
 
-makepad-widgets = { git = "https://github.com/jmbejar/makepad", branch = "moxin-release-v1" }
+makepad-widgets = { git = "https://github.com/noxware/makepad", rev = "7ebad2443" }
```

Comment:
This is a jmbejar's fork compatible branch with the widget added

https://github.com/noxware/makepad/tree/moxin-panel-for-jmbejar-remote

Reply:
Done!